api/services/dg-docassembly-api/dg-docassembly-api.ts

```diff
@@ -14,8 +14,7 @@ export function docAssemblyRoutes(app) {
 
     const urls = [
       '/api/form-definitions',
-      '/api/template-renditions',
-      '/health',
```

Comment:
wasnt russell looking into having a health check for em show?

Reply:
there is a healthcheck, it's in the application.ts in the root of the project. It collates all downstream APIs